http_server_spec/features/01_getting_started/method_not_allowed.feature

```diff
@@ -0,0 +1,9 @@
+@method-not-allowed @01-getting-started
+Feature: Method Not Allowed
+
+	@wip
+  Scenario: Finding GET for an endpoint with only HEAD
+    Given I make an GET request to "/head_request"
+    Then my response should have status code 405
+    And my response should have allowed headers of HEAD, OPTIONS
+    And my response should have an empty body
```

Comment:
What is up with the spacing on all these `@wip` additions? I see it on every single one.
```suggestion
  @wip
  Scenario: Finding GET for an endpoint with only HEAD
    Given I make an GET request to "/head_request"
    Then my response should have status code 405
    And my response should have allowed headers of HEAD, OPTIONS
    And my response should have an empty body
```

Reply:
I'm not sure why it spaced it like that but I fixed it!